### DISCUSSION TITLE
元宇宙到底是什麼?

### DISCUSSION THREAD

**Discussion by yuhao112** [1101-GS4719-C/Python-Community-GS4719-C#601] [Show and tell]
https://www.bnext.com.tw/article/67856/metaverse-dissect?
最近很流行的一個詞彙叫[元宇宙]，但他到底是什麼? 多數人只是有個共同的概念，只能簡單說出他是一個虛擬的世界，但裡面究竟是有甚麼，目前的科技確實還無法完成其中的想像，也希望他總有一天能真正地蓬勃發展

**Comment by handsomeniko** [comment]
推，謝謝分享

**Comment by jcpan821** [comment]
感覺跟VR很像

**Comment by Sazmafuji** [comment]
謝分享，

**Comment by Rosancl** [comment]
想了解元宇宙的話，可以看看《元宇宙: 全面即懂Metaverse的第一本書》

**Comment by 11timyyy** [comment]
Metaverse、NFT等等近幾年新興的詞彙，值得大家好好探索

**Comment by d26488399** [comment]
謝謝分享，有更了解了

**Comment by 110409538** [comment]
之前也一直對元宇宙有興趣，感謝分享

**Comment by leon10456** [comment]
感謝分享

**Comment by HsuChen88** [comment]
雖然我不是會員但感謝分享

**Comment by awkward-willy** [comment]
元宇宙的概念其實許多年前就有被提出
而最近才掀起一波風潮
在達成類似電影一級玩家那樣逼真的內容之前
還有許多層面的技術等著人類去開發
只能說可能還有一段時間才能達到那樣的境界吧

**Comment by sys511613** [comment]
現在有越來越多人去接觸區塊鏈、元宇宙、加密貨幣等
也有很多人在instagram或各個社群平台分享自己的經驗和帶領大家入門
推一個我朋友的instagram帳號：https://www.instagram.com/stevenhung.eth/

**Comment by jason79461385** [comment]
謝謝樓主分享，自己對元宇宙這塊比較沒有接觸，看完文章後對這些東西也更懂了些

**Comment by Lai27330** [comment]
謝謝分享，是最近炙手可熱的議題。

**Comment by alexthebigpp** [comment]
感覺離要實際做出來還有一段距離，現在市場上的元宇宙感覺大部分都是拿來行銷的手段而已
